pkg/transport/generic_transport_controller.go

```diff
@@ -847,8 +852,19 @@ func (c *genericTransportController) computeDestToCustomizedObjects(objectsToPro
 	return destToCustomizedObjects, bindingErrors
 }
 
+func TODO_FIXME_ADD_CONSTANT_CREATEONLY(obj *unstructured.Unstructured) Wrapee {
```

Comment:
FYI: The standard is to leave a comment that includes "TODO", which is followed by modern IDEs to list pending work.

Reply:
Resolved